src/main/java/com/toptrumps/db/PerformanceDAOImpl.java

```diff
@@ -0,0 +1,72 @@
+package com.toptrumps.db;
+import java.util.Map;
+import java.sql.*; 
+import com.toptrumps.core.statistics.GameStateCollector;
+import com.toptrumps.core.player.Player;
+
+public class PerformanceDAOImpl implements GameDAO{
+
+    private int gameID;
+    private Map<Player,Integer> map;
+
+    public PerformanceDAOImpl(GameStateCollector gameState){
+
+        this.gameID = getGameID();
```

Comment:
Do we need to get this id from the db with direct query, or we can just autoincrement it?

Reply:
So yes, it actually is auto-incremented. In the individual_game_data relation, the structure is just one row per game - however, in the player_performance relation, you can have more than one row per game played, which means it can't be auto-incremented in that relation. So this is what this does, it pulls the most recent game_id and associates it with all the most recent player performances.